week-2-practise/level-1/5-indexOf.js

```diff
@@ -3,7 +3,7 @@
 
 var itemToFind = 7;
 var arr = [3, 5, 61, 7, 123];
-var index; // ONLY EDIT THIS LINE
+var index = arr.indexOf(7); // ONLY EDIT THIS LINE
```

Comment:
Always use the variable that already holds the value you are checking! So in this case, using `itemToFind` helps your code be more readable